__tests__/cmds/openapi.test.js

```diff
@@ -555,7 +559,7 @@ describe('rdme openapi', () => {
         .post('/api/v1/api-specification', { registryUUID })
         .delayConnection(1000)
         .basicAuth({ user: key })
-        .reply(400, '<html></html>');
+        .reply(400, '<title>Application Error</title>');
```

Comment:
Before, we just assumed any ol' HTML was a timeout. Yikes! Now, we actually look for snippets of the Heroku/Cloudflare error pages.